### PR TITLE
Add ability to view all Tag Tabs

### LINKED ISSUE #7405: Have the bank tag tab list take the swap/insert rearrange mode into account
Currently you can only swap the tags' places, but inserting would be way more practical for ordering.
image
Letting go would make it so that tinderbox tab and mole tab would swap places, I'd like tinderbox be inserted above mole.

### LINKED ISSUE #9625: Bank tag tab tab
I suggest a bank tag tab tab, which would show all of your bank tag tabs' icons in a tab, to more easily locate them. Right now, if you have 20/30+ tag tabs, it takes a while to find the tag tab you want, scrolling through 3-4 at a time.

Having a tag tab tab (could do with a better name) would let me quickly click the tag tab I want.

Not sure about alternatives.

Not sure about where you'd place the button, maybe under the "+", or make that button "manage tag tabs" or something?

### PR DESCRIPTION
**Pull request by raiyni**
Closes #9625
Closes #7405

* Supports all operations from Tag Tabs
* Supports swap/insert mode
* Can be saved as a Tag Tab with `tagtabs` as the tab name

Gltlv5LA9G

### CODE REVIEW

**Comment by JeljoGb** [comment]
Is there a reason this isn't allowed to be in the client?

**Comment by JeljoGb** [comment]
Sorry adam I didn't mean it like that but I really want this

**Comment by JeljoGb** [comment]
On my end I only see "all checks passed", hence my confusion.
I mean no harm.

**Comment by raiyni** [comment]
Rebased bank script, added readable labels, and wrapped tab interface logic to manage the bank menu widgets.

File: runelite-client/src/main/java/net/runelite/client/plugins/banktags/tabs/TabInterface.java [hunk 1]
```diff
@@ -1035,6 +1162,12 @@ private Widget createGraphic(String name, int spriteId, int itemId, int width, i
 		return widget;
 	}
 
+	private Widget createGraphic(String name, int spriteId, int itemId, int width, int height, int x, int y, boolean hasListener)
+	{
+		return createGraphic(parent, name, spriteId, itemId, width, height, x, y, hasListener);
+
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/banktags/tabs/TabInterface.java]: Remove this extra newline

**Comment by AmazingJesus** [comment]
Is it possible to make the new worn items button to be replaced with the Runelite + button? Toggle maybe?

**Comment by raiyni** [comment]
#10575 solves that

**Pull request merged**